src/Hydration.js

```diff
@@ -19,6 +20,15 @@ class Hydration extends UserFitness {
   returnWeeklyNumOunces() {
     return this.userData.slice(-7).map(day => day.numOunces);
```

Comment:
It looks like you do not have the updated functions for when we changes the data functionality and how that was working on Friday - you will probable have to merge with master on your local side before pushing and merging this PR so that we do not lose all of that work.

Reply:
We have to do our merges on github. I can't merge my own PR's by pulling down the master and rewriting my section every time there is a merge conflict. You also don't lose work when you resolve a merge conflict if done correctly.